README.md

```diff
@@ -31,6 +31,92 @@ end
 
 Mox should be automatically started unless the `:applications` key is set inside `def application` in your `mix.exs`. In such cases, you need to [remove the `:applications` key in favor of `:extra_applications`](https://elixir-lang.org/blog/2017/01/05/elixir-v1-4-0-released/#application-inference) or call `Application.ensure_all_started(:mox)` in your `test/test_helper.exs`.
 
+## Basic Usage
+
+### 1) Add behaviour, defining the contract:
+```
```

Comment:
Use three backticks and `elixir` so that syntax gets highlighted (```elixir) 🙃

Reply:
✅ ❤️

Reply:
thanks @whatyouhide !